src/wyzeapy/payload_factory.py

```diff
@@ -36,7 +36,7 @@ def olive_create_post_payload(device_mac: str, device_model: str, prop_key: str,
         "did": device_mac,
         "model": device_model,
         "props": {
-            prop_key: str(value)
+            prop_key: value
```

Comment:
Not sure why original version use `str(value)` which turns `True` in to `"True"`, but we need `true` if it's a boolean type.

Reply:
I believe that the API at one time, needed it to be a string